loader/src/main/java/lucee/runtime/script/BaseScriptEngineFactory.java

```diff
@@ -52,7 +52,14 @@ public BaseScriptEngineFactory(final boolean tag, final int dialect) throws Serv
 				final String servletName = "";
 				final Map<String, Object> attributes = new HashMap<String, Object>();
 				final Map<String, String> initParams = new HashMap<String, String>();
-				final File root = new File("."); // working directory that the java command was called from
+				
+				// Allow override of context root
+				String rootPath = System.getProperty("lucee.cli.contextRoot");
+				if( rootPath == null ) {
```

Comment:
alway check for empty and not null -> if(!Util.isEmpty(rootPath))...

Reply:
@michaeloffner Done!